dbuf-lsp/src/diagnostic_handler/semantic_token/modifier.rs

```diff
@@ -0,0 +1,39 @@
+//! Module provides enum Modifier - enum of modifiers lsp uses to
+//! response to `textDocument/semantic` request.
+//!
+
+use tower_lsp::lsp_types::*;
+
+#[derive(Debug)]
+pub enum Modifier {
+    Declaration,
+}
+
+impl Modifier {
+    pub const COUNT: u32 = 1;
```

Comment:
You can use [`EnumCount`](https://docs.rs/strum_macros/latest/strum_macros/derive.EnumCount.html) from [`strum`](https://crates.io/crates/strum) crate. Or [`std::mem::variant_count`](https://doc.rust-lang.org/std/mem/fn.variant_count.html), but this requires nightly version of Rust.

Reply:
(`strum` might be better because it derives all the functionality you've written here by hand for you)`